changelog.md

```diff
@@ -1,7 +1,11 @@
+#1.1.1
```

Comment:
@dgwatkins, I have another fix waiting for approval (https://github.com/OnTheGoSystems/WPML-MU-Domain-Mapping/pull/4). So I suggest that we bump the version later when the 2 fixes are merged. Could you please remove this?